Documentation/General/productVision.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\begin{document}
+
+\title{Product Vision\\
+	\begin{small}
+	The why, who and what
+	\end{small}
+}
+\author{MoodCat.me}
+\maketitle
+
+\section{Why should we build MoodCat?}
+Listening to music is currently a solo activity.
+Various competitors provide a system that can offer users popular songs or songs that are similar to songs the user already listened to.
+These systems don't offer users songs they haven't heard yet and that might fit the user's current need.
+Since these systems provide solely songs that fit only the user and are very limited in varience, a better solution can be achieved.
+
+Secondly there are some competitors that offer users a room to socialize while experiencing songs selected by other users in the room.
+While this might offer the user unknown songs, the system is user-driven and does not offer the listeners that want to listen on their own.
```

Comment:
I think this introduction is still a bit blurry. The two core features should really excel here:

MoodCat will be the music platform of the future because:
- Users want social interaction. Current music listening platforms only show what music your friends are listening, if there is any user interaction at all. MoodCat allows users (and friends!) to unite in rooms, and listen to and discuss music together. With this we apply a successful feature from the live video/gaming streaming [[1](http://www.twitch.tv)] industry to the music industry.
- Music rooms never fall silent, instead they are constantly filled with relevant music which matches the current mood of the listeners in the room!